src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {StyleSheet, Text} from 'react-native';
+import {propTypes, defaultProps} from '../anchorForCommentsOnlyPropTypes';
+
+/*
+ * This is a default anchor component for regular links.
+ */
+const BaseAnchorForCommentsOnly = ({
+    href,
+    rel,
+    target,
+    children,
+    style,
```

Comment:
Should this include the `shouldDownloadFile` prop since it is being passed from `AnchorWithAuthToken.js `. If it is not necessary to have `shouldDownloadFile` here because it is passed in `...props`, then is it redundant when we have `shouldDownloadFile` in the props for `BaseAnchorForCommentsOnly/index.native.js `?

Reply:
This is a good question.

> Should this include the shouldDownloadFile prop since it is being passed from AnchorWithAuthToken.js

It's not necessary because it's not being used in this file. It _is_ passed to the nested `Text` in `...props`, but because the `Text` doesn't take a `shouldDownloadFile` prop it just has no effect.

> is it redundant when we have shouldDownloadFile in the props for BaseAnchorForCommentsOnly/index.native.js

It's not redundant, because the `shouldDownloadFile` prop is used to determine `onPress` prop of the `Text` component in `BaseAnchorForCommentsOnly/index.native.js`. Then it's also passed to the `Text` component as part of `...props`, but like the web implementation just has no effect.